bower.json

```diff
@@ -46,6 +46,6 @@
   },
   "devDependencies": {
     "angular-mocks": "1.3.0 - 1.5.*",
-    "angular-ui-router": "^0.3.2"
+    "angular-ui-router": "1.0.0-beta.3"
```

Comment:
Moreover, I think that 1.0.0-beta should be api-compatible with what will be 1.0.0 release.